lib/package.gd

```diff
@@ -159,6 +159,16 @@ DeclareGlobalFunction( "RECORDS_FILE" );
 DeclareGlobalFunction( "SetPackageInfo" );
 
 
+#############################################################################
+##
+#F  AddPackageInfo( files, pkgdir, ignore, metapackages... )
+##
+##  Adds package info records from a file list to the global list
+##  of info records.
+##
+DeclareGlobalFunction( "AddPackageInfos" );
+
```

Comment:
The comment says `AddPackageInfo` without `s`. Also, it's not clear to me what is a "file list". It it a list of files? Or a file containing a list? Or something else? What's the use case for this function?